tests/test_library.py

```diff
@@ -5,6 +5,7 @@
 import pykka
 from mopidy import core
 from mopidy.models import SearchResult, Track
+from unittest import mock
```

Comment:
This import should be in the first block, with the other imports from the stdlib.

Reply:
Didn't we have something that used to check import order? `isort` or something? Or is flake8 not supposed to catch this?

Reply:
We do seem to be missing a pyproject.toml in this repo. Not that it seems to make any difference

Reply:
```
flake8.style_guide        MainProcess    571 DEBUG    Deciding if "I100" should be reported
flake8.style_guide        MainProcess    571 DEBUG    The user configured "I100" to be "Ignored.Implicitly", "Selected.Implicitly"
flake8.style_guide        MainProcess    571 DEBUG    "I100" will be "Decision.Ignored"
```
.... why?